runtime/wasm/src/host.rs

```diff
@@ -285,13 +285,13 @@ impl RuntimeHost {
 
     fn matches_block_trigger(&self, block_trigger_type: &EthereumBlockTriggerType) -> bool {
         let source_address_matches = match block_trigger_type {
-            EthereumBlockTriggerType::WithCallTo(address) => {
+            EthereumBlockTriggerType::WithCallTo(address, _block_type) => {
```

Comment:
No variables would have to be ignored if the separation of types described above is implemented.